Localization/en-US/Mods.PathOfTerraria.Affixes.hjson

```diff
@@ -34,7 +34,7 @@ BuffPoisonedHitsAffix.Description: "{1}{0}% damage to poisoned enemies"
 MovementSpeed.Description: "{1}{0}% movement speed"
 HealOnKillingBurningEnemiesAffix.Description: "{1}{0} health gained on killing a burned enemy"
 IncreasedAttackSpeedAffix.Description: "{1}{0}% faster attack speed"
-AddedAttackSpeedAffix.Description: "{1}{0} reduced attack speed"
+AddedAttackSpeedAffix.Description: "{1}{0}% increased attack speed"
```

Comment:
This may require some additional followup work to make this dynamic. There may be a case where it goes into the negatives, and the text dynamically changes based on positive or negative values.

I think this is fine for now however.

Reply:
Technically it's possible to make a "negative increase" so to speak in the current system, but I don't think we really have a use for "+x decreased y" over "-x increased y" - especially for the playtest.